common/classesiconsindex_data.cpp

```diff
@@ -0,0 +1,139 @@
+// This is machine generated code, DO NOT EDIT.
+// This generated file is versionned only to easier crossbuilding to others architectures.
+// See classesiconsextractor.cpp to change this code.
+namespace GammaRay {
+namespace ClassesIconsIndex {
+static QHash<int, QString> s_numberToKey;
+static QHash<QString, int> s_keyToNumber;
+
+static void initializeClassesIconsIndex()
+{
+    Q_ASSERT(s_numberToKey.isEmpty());
+    Q_ASSERT(s_keyToNumber.isEmpty());
+    s_numberToKey[0] = QLatin1String(":/gammaray/icons/ui/classes/QQuickBorderImage/default.png");
+    s_keyToNumber[s_numberToKey[0]] = 0;
```

Comment:
one way might be to just send the complete map from the probe to the client, that's how we solve this for enums for example